pkgs/tools/misc/goreleaser/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "goreleaser";
+  version = "0.145.0";
+
+  src = fetchFromGitHub {
+    owner = "goreleaser";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1w2mbyywr3zsn068cshkx502x0zrxrbrgaw23x4spfri0nk6v4fd";
+  };
+
+  vendorSha256 = "0drk58bhcvx75cd6s0xnyh6swph1vlvpzp2nngr7agvjdcrbady6";
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+    "-X main.builtBy=nixpkgs"
+  ];
+
+  doCheck = false;
```

```suggestion
  # tests expect the source files to be a build repo
  doCheck = false;
```

Or something similar.

---

@SuperSandro2000 ✅